### PR TITLE
feat: best/worst_season methods/tests

### PR DESCRIPTION
**Pull request by Jonathan-Atkins**

### CODE REVIEW

File: lib/team_statistics.rb [hunk 1]
```diff
@@ -128,19 +130,85 @@ def biggest_team_blowout(team_id)
...
+    end
+    total_games
+  end
+
+
+  def worst_season(team_id)
+    season_stats = Hash.new {|hash,key| hash[key] = {games_played: 0, wins: 0}}
+    @games.each do |game|
+      next unless game.away_team_id == team_id || game.home_team_id == team_id
+      season = game.season
```
Comment (sethverrill) [inline lib/team_statistics.rb:186]: I like how this turned out. Great job!

**Review by sethverrill — APPROVED** [review]
I think this looks really good.

**Comment by MiTOBrien** [comment]
Great work Jonathan.

**Comment by sethverrill** [comment]
The merge conflict was basically a formatting issue. Lots of extra spaces and tabs. I _think_ I got it handled, but double check your ends when you pull.

**Comment by Jonathan-Atkins** [comment]
thanks all !

**Pull request merged**